src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,13 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(Comparator.comparingInt(String::length).reversed())
+                .distinct()
```

Comment:
```suggestion
                .distinct()
                .sorted(Comparator.comparingInt(String::length).reversed()) 
```
스트림의 실행되는 순서를 변경해보면 어떨까요?
스트림은 정의된 순서에 따라 실행되기 때문에 어떤 순서로 일을 시키느냐도 중요해요. :)